emoji/src/components/EmojiContainer.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import EmojiRow from './EmojiRow';
+
+class EmojiContainer extends Component {
+    render() {
```

Comment:
If your react component has only `render` method, write it as a pure function: `const EmojiContainer = () => <EmojiRow /> ` . Just one line would replace the whole code. Furthermore, functions are better in performance